shomei/validators.py

```diff
@@ -69,3 +69,25 @@ def validate_github_token(token):
         console.print("[dim]continuing anyway, but double-check if you get auth errors[/dim]\n")
 
     return True, None
+
+def validate_github_username(username):
+    """
+    Validate whether the given username exists on GitHub.
+
+    Args:
+        username: The GitHub username to validate
+
+    Returns:
+        tuple: (exists: bool, error_message: str or None)
+
+    Notes:
+        Print a warning if given username doesn't exist but don't block.
+    """
+    
```

Comment:
should probably add an empty/whitespace check before hitting the API, same pattern as validate_repo_name does. (just in case) 

something like:

```suggestion

if not username or len(username.strip()) == 0:
    return False, "username cannot be empty"
    
username = username.strip()
```